packages/react-components/src/components/VideoGallery/useVideoTileContextualMenuProps.ts

```diff
@@ -165,6 +168,21 @@ export const useVideoTileContextualMenuProps = (props: {
         });
       }
     }
+    /* @conditional-compile-remove(soft-mute) */
+    if (onMuteParticipant) {
+      items.push({
+        key: 'mute',
+        text: 'Mute',
+        iconProps: {
+          iconName: participant.isMuted ? 'ContextualMenuMicMutedIcon' : 'ContextualMenuMicUnmutedIcon',
```

Comment:
I think this ternary should flipped here because of the snapshots:
`iconName: participant.isMuted ? 'ContextualMenuMicUnmutedIcon' : 'ContextualMenuMicMutedIcon'`

Reply:
Changed so that it only shows the ContextualMenuMicMutedIcon. There's no reason to change the icon as we don't allow unmuting participants at this time.